src/WWT.Providers/WttContext.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Web;
+
+namespace WWT.Providers
+{
+    public readonly struct WttContext
```

Comment:
Is this intentionally `WttContext` and not `WwtContext`?

Reply:
Oops - accidental. I'll get that fixed. WTT was a product I used a few years ago so I keep typing that without realizing it.